modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMQContainer.java

```diff
@@ -0,0 +1,358 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static java.lang.String.join;
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toSet;
+
+/**
+ * Testcontainer for RabbitMQ.
+ *
+ * @author Martin Greber
+ */
+public class RabbitMQContainer<S extends RabbitMQContainer<S>> extends GenericContainer<S> {
```

Comment:
Since this class is intended to be used directly, it should not provide self-typing signature (P.S. we plan to remove it all together soon-ish)
```suggestion
public class RabbitMQContainer extends GenericContainer<RabbitMQContainer> {
```